Dockerfile-linux.template

```
@@ -383,6 +383,9 @@ RUN set -eux; \
 		--enable-zts \
 # https://externals.io/message/118859
 		--disable-zend-signals \
+{{ if ((.version | version_id) >= ("8.2.4" | version_id) and (.version | version_id) < ("8.3" | version_id)) or ((.version | version_id) >= ("8.1.17" | version_id) and (.version | version_id) < ("8.2" | version_id)) then ( -}}
```

Since our supported versions of 8.1 and 8.2 are already new enough, I think we can drastically simplify this conditional: :sweat_smile:

```suggestion
{{ if [ "8.1", "8.2" ] | index(env.version | rtrimstr("-rc")) then ( -}}
```

(Happy to take over from here if you'd prefer :bow: :heart:)

---

Good catch! Don't hesitate to take over!